lib/apiClient.ts

```diff
@@ -0,0 +1,8 @@
+import axios from 'axios';
+import { BASE_URL } from './constants';
+
```

Comment:
We need to make an axios instance like https://axios-http.com/docs/instance and then call HTTP requests on the instance

Reply:
why would an axios instance be preferred over the way we implemented (including a url in the put method)? it def looks cleaner tho lol

Reply:
In fact, just do what JPAL does: https://github.com/Code-4-Community/jpal-frontend/blob/master/src/api/apiClient.ts

Reply:
gotcha thanks Varun!